rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -184,6 +186,12 @@ def _add_builtin_decoders(self, rules: DecodingRules) -> None:
         self._add_single_decoder(class_name='Oneinchv5', decoder_class=Oneinchv5Decoder, rules=rules)  # noqa: E501
         self._add_single_decoder(class_name='SocketBridgeDecoder', decoder_class=SocketBridgeDecoder, rules=rules)  # noqa: E501
 
+        # Excluding Gnosis and Polygon PoS because they dont have ETH as native token
+        if self.evm_inquirer.chain_id not in CHAINS_WITHOUT_NATIVE_ETH:
+            self._add_single_decoder(
+                class_name='WethDecoder', decoder_class=WethDecoder, rules=rules,
+            )
```

Comment:
when splitting the arguments we use a new line for each one

```suggestion
            self._add_single_decoder(
                class_name='WethDecoder',
                decoder_class=WethDecoder,
                rules=rules,
            )
```